File: client/task-master/src/app/models/Card.ts
import { CardAttachment } from './CardAttachment';
import { CardComment } from './CardComment';

// Объект карточки
export class Card {
  // Идентификатор карточки
  id: string;
  // Флаг наличия обложки карточки
  hasCoverImage: boolean;
  // URL изображения обложки карточки
  imageUrl: string;
  // Заголовок карточки
  title: string = '';
  // Описание карточки
  description: string = '';
  // Вложения карточки
  attachments: CardAttachment[];
}

File: server/TaskMaster/TaskMaster.DataWebApi/Models/Card.cs
using TaskMaster.DataAccessModule.Models;

namespace TaskMaster.DataWebApi.Models
{
	/// <summary>
	/// Карточка на доске.
	/// </summary>
	public class Card
	{
		/// <summary>
		/// Идентификатор карточки.
		/// </summary>
		public Guid Id { get; set; }

		/// <summary>
		/// Заголовок карточки.
		/// </summary>
		public string Title { get; set; } = null!;

		/// <summary>
		/// Описание карточки.
		/// </summary>
		public string? Description { get; set; }

		/// <summary>
		/// Указывает, есть ли у карточки обложка.
		/// </summary>
		public bool HasCoverImage { get; set; }

		/// <summary>
		/// Список вложений к карточке.
		/// </summary>
		public List<CardAttachment>? Attachments { get; set; }
	}

}
